src/cart/crt_swim.c

```diff
@@ -67,18 +68,26 @@ crt_proc_struct_swim_member_update(crt_proc_t proc,
 	return 0;
 }
 
-CRT_RPC_DECLARE(crt_rpc_swim, CRT_ISEQ_RPC_SWIM, CRT_OSEQ_RPC_SWIM)
-CRT_RPC_DEFINE(crt_rpc_swim, CRT_ISEQ_RPC_SWIM, CRT_OSEQ_RPC_SWIM)
+CRT_RPC_DECLARE(crt_rpc_swim, CRT_ISEQ_RPC_SWIM, /* empty */)
+CRT_RPC_DEFINE(crt_rpc_swim,  CRT_ISEQ_RPC_SWIM, /* empty */)
+
+CRT_RPC_DECLARE(crt_rpc_swim2, CRT_ISEQ_RPC_SWIM, CRT_OSEQ_RPC_SWIM)
+CRT_RPC_DEFINE(crt_rpc_swim2,  CRT_ISEQ_RPC_SWIM, CRT_OSEQ_RPC_SWIM)
```

Comment:
Could crt_rpc_swim2 have a better name? Or, would it make any sense to at least have a comment explaining the purposes of crt_rpc_swim and crt_rpc_swim2, respectively?

Reply:
Ok. Will think how to change this...